### ISSUE TITLE
/*beautify ignore*/ not working in css

### ISSUE THREAD

**Issue by Nani-Sore-0** [victorporof/Sublime-HTMLPrettify#588]
Before beautify:
```
  /* beautify ignore:start */
  grid-template-areas:
    "nav-banner nav-banner nav-banner nav-close-button"
    "nav-uros nav-milos nav-markovic nav-matija"
    "nav-djole nav-mihajlo nav-srdjan .";
 /* beautify ignore:end */
```
After beautify: 
```
  /* beautify ignore:start */
  grid-template-areas: "nav-banner nav-banner nav-banner nav-close-button" "nav-uros nav-milos nav-markovic nav-matija" "nav-djole nav-mihajlo nav-srdjan .";
  /* beautify ignore:end */
```
I tried /* beautify ignore:start */ as well it also dose not work. Am I doing something wrong?

**Comment by bitwiseman** [comment]
@Nani-Sore-0 This is a limitation of the css beautifier. https://github.com/beautify-web/js-beautify/issues/813

**Comment by victorporof** [comment]
Should be fixed now! Update to 2.2.0 https://github.com/victorporof/Sublime-HTMLPrettify/releases

If not, please send a feature request to https://github.com/beautify-web/js-beautify/issues
